assembl/static2/js/app/utils/translations.js

```diff
@@ -1324,6 +1325,7 @@ const Translations = {
         numberOfResponses_1: "%{count} response",
         postSuccessMsg: "Thanks for your participation. Your fiction has been published successfully!",
         sideCommentSuccessMsg: "Your comment has been published successfully!",
+        sideCommentEditSuccessMsg: "Your comment has been edited successfully!",
```

Comment:
rather ```successfully edited``` no ? @ayyazdaniaryan ? This sounds a bit like frenchized english 🇫🇷 but I might be wrong

Reply:
Both seems correct grammatically but the emphasis is put on the last word so in our case, we emphase the successfulness :)

Reply:
Allright je vois :)